lib/l10n/app_en.arb

```diff
@@ -671,6 +667,10 @@
   "@postViewType": {
     "description": "Label for the post view type (compact/card) in Setting -> Appearance -> Posts"
   },
+  "postingRestrictedToMods": "Posting restricted to mods",
```

Comment:
We do have this string as well!

https://github.com/thunder-app/thunder/blob/c16848ba2768308fb4956c99ac6bfc86ecdbd853/lib/l10n/app_en.arb#L594